zerver/openapi/zulip.yaml

```diff
@@ -1050,6 +1050,43 @@ paths:
                         "result": "success",
                         "uri": "/user_uploads/1/4e/m2A3MSqFnWRLUf9SaPzQ0Up_/zulip.txt"
                     }
+  /user_uploads/{realm_id_str}/{filename}:
+    get:
+      description: Get a temporary URL for access to the file that doesn't require authentication.
```

Comment:
Stylistically, we're now doing `description:` always with just a `|` on the first line with the body line-wrapped; it's a better editing experience that way.